framework/web/Response.php

```diff
@@ -431,7 +431,7 @@ protected function sendContent()
         }
 
         if (function_exists('set_time_limit')) {
-            set_time_limit(0); // Reset time limit for big files
+            @set_time_limit(0); // Reset time limit for big files
         } else {
             Yii::warning('set_time_limit() is not available', __METHOD__);
         }
```

Comment:
I suggest:

```php
        // try to reset time limit for big files
        if (!@set_time_limit(0)) {
            Yii::warning('set_time_limit() is not available', __METHOD__);
        }
```

Reply:
Good point, should be a warning. Done now (07e8a74d219ff6334cf75a6915e18208bcad4337).